tests/foreman/cli/test_syncplan.py

```diff
@@ -273,3 +314,102 @@ def test_verify_bugzilla_1261122(self):
         new_sync_plan = self._make_sync_plan()
         result = SyncPlan.info({'id': new_sync_plan['id']})
         self.assertIsNotNone(result.get('enabled'))
+
+    @tier2
+    def test_positive_synchronize_product(self):
+        """Create a sync plan with one product and sync it automatically.
+
+        @Assert: Product is synchronized successfully.
+
+        @Feature: SyncPlan
+        """
+        org = make_org()
```

Comment:
Is it specific reason to create org per each test? It seems as a redundancy for me